plot_pixel_values.py

```diff
@@ -22,24 +24,70 @@ def get_opt():
     parser.add_argument('--fit-scaling',
                         action='store_true',
                         help='Fit the dark and scale factor (default=False)')
-
+    parser.add_argument('--plot-fits',
+                        action='store_true')
     parser.add_argument('--n-brightest',
-                        default=128,
+                        default=64,
                         type=int,
-                        help='Plot the N brightest (either 128 or n**2)')
+                        help='Plot the N brightest (either 64 or n**2)')
 
     args = parser.parse_args()
     return args
 
 opt = get_opt()
 
+
+def scale_model(pars, t_ccd):
+    scale, y_const, x_const = pars
+    return 1 / np.exp(np.log(scale) / 4.0 * (t_ccd - x_const))
```

Comment:
Also, if we want scale to be ~0.70, I still think we need the reciprocal or "dark_t_ref / np.exp ".